docs/source/blueprints/BP000.rst

```diff
@@ -0,0 +1,307 @@
+
+ * Motivation
+
+ * Specification
+
+ * Backwards Compatibility
+
+ * Security Implications
+
+ * How to Teach This
+
+ * Related Issues
+
+ * References
+
+Bellow a brief description of what you should write in each section:
+
+ * TL;DR: Should be a short description of your blueprint. Like an abstract. We
+   recommend writing this at the end of your first draft. This will give you a
+   better overview of it.
+
+ * Motivation: This should be the motivation of your proposed solution, not the
+   motivation of the blueprint itself. It describes the problem. Here, the
+   proposed solution should answer “why” this is needed.
+
+ * Specification: In this section, you should describe how you are going to
+   solve the problem. You can create subsections here to organize your ideas
+   better. Please keep in mind that it is useful to mention the details, with
+   code snippets, examples, and/or references. This will save you time, making
+   sure that everyone is in agreement with the proposed solution.
+
+ * Backwards Compatibility: How is your proposal going to affect older versions
+   of Avocado? Should we deprecate some modules, classes, or methods? Are we
+   going to keep backwards compatibility or not?
+
+ * Security Implications: What are your concerns about security with your
+   proposed solution?
+
+ * How to Teach This: What is the best way to teach our devs and users your new
+   feature/solution?
+
+ * Related Issues: Here, you should mention Github links for both: a) current
+   open issues that are blocking waiting for your BP and b) all open issues
+   that will make this BP as “implemented” when closed.
+
+   * Issues to address this BP
+
+   * Issues this BP will solve
+
+ * References: Any external reference for helping understand the problem and
+   your solution.
+
+Backwards Compatibility
+***********************
+
+So far, we are on our 3rth blueprint (BP003 was the last one). This BP000
+should have been released before those blueprints. So probably those three
+blueprints are not 100% compliance with this meta blueprint, and that is fine.
+We were learning on the fly. We don’t need to change any of those blueprints
+after BP000 gets approved.
+
+Security Implications
+*********************
+
+No security implications found so far.
+
+How to Teach This
+*****************
+
+Getting used to writing blueprints is not an easy task. And probably we are
+going to find non-planned issues with this process on the way.  The general
+rule of thumb is to use common sense. To make this more public, we could:
+
+ * If approved, BP000 should be on top of our blueprints lists for reference.
+
+ * We could also have a template inside the `blueprints` directory to help
+   people when submitting their own blueprints.
+
+ * Also, we could include pointers and instructions in our development guide
+   for this BP.
+
+ * Another good practice would be to make comments in Avocado’s source code
+   with some pointers to specific blueprints. 
```

Comment:
I see. Since we are still learning how to make blueprints, I guess the language needs to change from draft to approved. During the draft phase, the author is trying to prove something, and after that, we are informing our users/developers.

But I guess the whole idea of this section is to list all things that we need to do, in order to teach our users about this BP.

Reply:
Alright, both ways work fine and we can decide which form is better in the future with some extra use.